molgenis-questionnaires/src/main/frontend/src/components/ChapterForm.vue

```diff
@@ -9,6 +9,20 @@
   </form-component>
 </template>
 
+<style>
+  .questionnaire-chapter .pl-2 > fieldset > small {
+    font-size: 100%;
+  }
+
+  .questionnaire-chapter [class^="pl-"] > fieldset > legend {
+    padding-top: 1rem;
+  }
+
+  .questionnaire-chapter .pl-4 > fieldset > legend {
```

Comment:
if ' .questionnaire-chapter .pl-2 > fieldset > small' and 'questionnaire-chapter .pl-4 > fieldset > legend ' get the same style u can use the union selector 

 .questionnaire-chapter .pl-2 > fieldset > small,
 .questionnaire-chapter .pl-4 > fieldset > legend {
       font-size: 100%;
  }`
`